dgm-builder/src/main/java/com/cloudbees/groovy/cps/tool/Translator.java

```diff
@@ -95,7 +95,7 @@
 import java.util.Optional;
 import java.util.Set;
 import java.util.TreeMap;
-import javax.annotation.Generated;
+import javax.annotation.processing.Generated;
```

Comment:
IDK what the exact history is here, but this class is now an exported class in `java.compiler` and the source code is the same. The `RetentionPolicy` is `SOURCE`, so I think that changing the class is fine, but I'm not sure.

Reply:
Turns out that it is actually simpler to depend on `javax.annotation-api` and use the old package for now while we are running `dgm-builder` against Java 11 but compiling and running `groovy-cps` (and `workflow-cps` once we merge this into it) for compatibility with Java 8 to avoid needing to specify a bunch of pom overrides.